packages/perseus/src/widgets/radio/widget.jsx

```diff
@@ -443,28 +475,35 @@ class Radio extends React.Component<Props> {
             // Support legacy choiceState implementation
         }
         /* istanbul ignore if - props.values is deprecated */
-        // $FlowFixMe[prop-missing]
         if (props.values) {
             let noneOfTheAboveIndex = null;
             let noneOfTheAboveSelected = false;
 
-            const values = props.values.slice();
+            const choicesSelected = props.values.map((value) => !!value);
```

Comment:
**style suggestion (non-blocking):** `props.value.map(Boolean)` is a shortcut for this.

Reply:
Ooo, this is awesome! Thanks for pointing this out! Just updated to the shortcut you shared. I hope I remember this in the future as it is much cleaner looking :)